### DISCUSSION TITLE
Thoughts on adding a background to the modal window?

### DISCUSSION THREAD

**Discussion by GeorgeyB** [payloadcms/payload#617] [Feature Requests & Ideas]
Sometimes the lack of opaque background on the modal window causes an accessibility issue due to the text/content behind - it can be a little hard on the eyes:

![image](https://user-images.githubusercontent.com/13889255/171021400-07fc9a0d-b689-426c-9f98-a840f217c27a.png)

I appreciate the simple aesthetic of Payload _a lot_ but I think giving the modal window a white background would be a dramatic improvement without taking away any of the beauty:

![image](https://user-images.githubusercontent.com/13889255/171021849-f621ff24-e51d-4161-b430-032dccea06dd.png)

I know I can (and have) add this change to my project's stylesheet, but figured it might be a welcome change for Payload itself.

P.S. the modal would also benefit from focus trapping.

**Comment by jmikrut** [comment]
Hey @GeorgeyB these are great suggestions.

We're working through some admin-wide UX improvements right now and we'll factor this in too!
